.ruby-version

```diff
@@ -1 +1 @@
-2.7.1
+2.7.2
```

Comment:
@jcoyne you added this file in the initial commit. Do we want .ruby-version committed or nah? I can amend the PR if not.

Reply:
I think rails generates this by default. But I think it'll just give us headaches when it doesn't match what we have on the server. I'd ditch it.

Reply:
Removing